package.json

```diff
@@ -20,6 +20,18 @@
     "type": "git",
     "url": "git+https://github.com/MetaMask/rpc-cap.git"
   },
+  "dependencies": {
```

Comment:
Why did this whole block move? It’s hard to review dep changes when all lines change.

Reply:
It won't move again, I just want `dependencies` before `devDependencies`.